.github/workflows/container.yml

```diff
@@ -17,10 +17,10 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
We were getting warnings about these actions still being Node 12, I've done a driveby version bump for them.